Firestore/core/src/firebase/firestore/local/leveldb_key.h

```diff
@@ -235,7 +237,8 @@ class LevelDbMutationQueueKey {
    * returned, this instance is in an undefined state until the next call to
    * `Decode()`.
    */
-  bool Decode(leveldb::Slice key);
+  ABSL_MUST_USE_RESULT
+  bool Decode(absl::string_view key);
```

Comment:
There are more Decodes in this file and this seems like a generally good idea.

Reply:
Done.